x-pack/test/security_solution_api_integration/config/serverless/config.base.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrConfigProviderContext } from '@kbn/test';
+
+export interface CreateTestConfigOptions {
+  testFiles: string[];
+  junit: { reportName: string };
+}
+
+export function createTestConfig(options: CreateTestConfigOptions) {
+  return async ({ readConfigFile }: FtrConfigProviderContext) => {
+    const svlSharedConfig = await readConfigFile(
+      require.resolve('../../../../test_serverless/shared/config.base.ts')
+    );
+
```

Comment:
I'm wondering if there could be a reason for reusing `x-pack/test_serverless/functional/test_suites/security/config.ts` instead of `test_serverless/shared/config.base.ts`. Probably not but wanted to check what are your thoughts.

Reply:
Yeah sure, that's a valid point!
We add one important additional field for the CI which is the `mochaOptions` to pass our own allowed/skipped mocha tags, so we get everything defined under the `base.config.ts` and we just tweak a small configuration